### ISSUE TITLE
Plugin crashes at configure time at GoLand v2023.2.2

### ISSUE THREAD

**Issue by mzampetakis** [cytechmobile/radicle-jetbrains-plugin#429]
Using Goland : 
```
GoLand 2023.2.2
Build #GO-232.9921.53, built on September 14, 2023
Runtime version: 17.0.8+7-b1000.22 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.1.1
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 10
Metal Rendering is ON
Registry:
    editor.focus.mode.color.light=415967
    ide.editor.tab.selection.animation=true
    suggest.all.run.configurations.from.context=true
    ide.animate.toolwindows=true
    ide.completion.variant.limit=500
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=415967

Non-Bundled Plugins:
    com.intellij.javafx (1.0.4)
    String Manipulation (9.12.0)
    org.toml.lang (232.8660.88)
    intellij.vitejs (232.8660.149)
    com.mallowigi (93.1.0)
    zielu.gittoolbox (500.0.23+222)
    intellij.prettierJS (232.9921.53)
    com.chrisrm.idea.MaterialThemeUI (8.12.6)
```

I install the radicle plugin v0.7.3 and as soon as I press the configure link in the pop up it crashes.
The error report is the folloing: 
```
   Cannot create configurable
   
   com.intellij.diagnostic.PluginException: Cannot create class network.radicle.jetbrains.radiclejetbrainsplugin.config.RadicleSettingsView (classloader=PluginClassLoader(plugin=PluginDescriptor(name=Radicle, id=network.radicle.jetbrains, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/GoLand2023.2/plugins/radicle-jetbrains-plugin, version=0.7.3, package=null, isBundled=false), packagePrefix=null, state=active))
	at com.intellij.serviceContainer.ComponentManagerImpl.doInstantiateClass(ComponentManagerImpl.kt:1003)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:954)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClass(ComponentManagerImpl.kt:1022)
	at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:429)
	at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:338)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.createConfigurable(ConfigurableWrapper.java:39)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:116)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:95)
	at com.intellij.ide.actions.ShowSettingsUtilImpl$showSettingsDialog$3.invoke(ShowSettingsUtilImpl.kt:116)
	at com.intellij.ide.actions.ShowSettingsUtilImpl$showSettingsDialog$3.invoke(ShowSettingsUtilImpl.kt:116)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog$lambda$2(ShowSettingsUtilImpl.kt:116)
	at com.intellij.ide.actions.ShowSettingsUtilImpl$showSettingsDialog$config$1.accept(ShowSettingsUtilImpl.kt:130)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.test(ConfigurableVisitor.java:23)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.test(ConfigurableVisitor.java:18)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:57)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:64)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:46)
	at com.intellij.openapi.options.ex.ConfigurableVisitor.find(ConfigurableVisitor.java:30)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:132)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:116)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.showSettingsDialog(ShowSettingsUtilImpl.kt:109)
	at network.radicle.jetbrains.radiclejetbrainsplugin.actions.rad.RadAction$ConfigureRadCliNotificationAction.actionPerformed(RadAction.java:315)
	at com.intellij.notification.NotificationAction.actionPerformed(NotificationAction.java:33)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:339)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:313)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:362)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:313)
	at com.intellij.openapi.fileEditor.impl.IdeUiServiceImpl.performActionDumbAwareWithCallbacks(IdeUiServiceImpl.java:114)
	at com.intellij.notification.Notification.fire(Notification.java:283)
	at com.intellij.notification.impl.NotificationsManagerImpl.lambda$createAction$17(NotificationsManagerImpl.java:919)
	at com.intellij.ui.components.labels.LinkLabel.doClick(LinkLabel.java:175)
	at com.intellij.ui.components.labels.LinkLabel.doClick(LinkLabel.java:389)
	at com.intellij.ui.components.labels.LinkLabel$MyMouseHandler.mouseReleased(LinkLabel.java:362)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6657)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
	at java.desktop/java.awt.Component.processEvent(Component.java:6422)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4954)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4581)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4522)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2808)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:794)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:739)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:733)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:766)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:763)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:685)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:633)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:588)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:992)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:992)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
Caused by: java.lang.NoClassDefFoundError: org/apache/maven/artifact/versioning/ArtifactVersion
	at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.base/java.lang.Class.privateGetDeclaredConstructors(Class.java:3373)
	at java.base/java.lang.Class.getDeclaredConstructors(Class.java:2555)
	at com.intellij.serviceContainer.ComponentManagerImpl.doInstantiateClass(ComponentManagerImpl.kt:968)
	... 77 more
Caused by: java.lang.ClassNotFoundException: org.apache.maven.artifact.versioning.ArtifactVersion PluginClassLoader(plugin=PluginDescriptor(name=Radicle, id=network.radicle.jetbrains, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/GoLand2023.2/plugins/radicle-jetbrains-plugin, version=0.7.3, package=null, isBundled=false), packagePrefix=null, state=active)
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.kt:156)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:525)
	... 81 more
```

**Comment by mzampetakis** [comment]
Feel free to close this, as the [v0.8.0](https://github.com/cytechmobile/radicle-jetbrains-plugin/releases/tag/v0.8.0) release version of the plugin has resolved this.

**Comment by JChrist** [comment]
Closed via #452
